static/scripts/fix-redirect.js

```diff
@@ -971,6 +971,9 @@ const redirects = [
 		fromExact("/docs/guides/other-guides/load-balancing-kubernetes-clusters/"),
 		"/docs/k8s/load-balancing/load-balancing-kubernetes-clusters/",
 	],
+		[
+		fromExact("/docs/universal-gateway/cloud-endpoints/quickstart/"),
```

Comment:
Be sure to do a search in the docs repo for `universal-gateway/cloud-endpoints/quickstart` so you can replace the outdated links. Even if you add a redirect, docusaurus does client-side navigation, so links from within the docs site will 404